src/App.test.tsx

```diff
@@ -1,11 +1,45 @@
 import { render, waitFor } from "@testing-library/react";
 import App from "./App";
+import * as React from "react";
+import fetch from "node-fetch";
+
+jest.mock("@deskpro/app-sdk", () => ({
+  ...jest.requireActual("@deskpro/app-sdk"),
+  useDeskproAppEvents: (
+    hooks: { [key: string]: (param: Record<string, unknown>) => void },
+    deps: [] = []
+  ) => {
+    const deskproAppEventsObj = {
+      data: {
+        ticket: {
+          id: 1,
+          subject: "Test Ticket",
+        },
+      },
+    };
+    React.useEffect(() => {
+      !!hooks.onChange && hooks.onChange(deskproAppEventsObj);
+      !!hooks.onShow && hooks.onShow(deskproAppEventsObj);
+      !!hooks.onReady && hooks.onReady(deskproAppEventsObj);
+      /* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Need to remove `/* eslint-disable react-hooks/exhaustive-deps */` or somehow add an exception to eslint only for these mocks.

Reply:
Forgot to add next line